### PR TITLE
Emulator cluster firmware style

### PR DESCRIPTION
**Pull request by SClarkPhysics**
This PR changes the TrackJet Emulation Clustering algorithm to more closely match what is done in firmware. Specifically, it changes the way the binning is done in the ηφ plane. We also change the python config file so that the max eta is appropriately set, which is used in the APx workflow

This has already been merged to cms-l1t-offline:phase2-l1t-integration-13_3_0_pre3 via [PR 1170](https://github.com/cms-l1t-offline/cmssw/pull/1170) . See discussion there

### CODE REVIEW

**Comment by epalencia** [comment]
Please test

**Comment by aloeliger** [comment]
+l1
- reviewed by L1 here: https://github.com/cms-l1t-offline/cmssw/pull/1170

**Comment by srimanob** [comment]
+Upgrade

File: L1Trigger/L1TTrackMatch/plugins/L1TrackJetClustering.h [hunk 1]
```diff
@@ -17,6 +17,22 @@ namespace l1ttrackjet {
   const unsigned int ETA_INTPART_BITS{3};
   const unsigned int kExtraGlobalPhiBit{4};
 
+  static constexpr int kEtaWordLength = 15;
+  static constexpr int kPhiWordLength = 12;
+
+  //Constants used for jet clustering in eta direction
+  static constexpr int kThirteenBitMask = 8191;  // 0001111111111111 (bin) = 8191 (dec)
```
Comment (rappoccio) [inline L1Trigger/L1TTrackMatch/plugins/L1TrackJetClustering.h:24]: Could these be made into binary literals? Then you don't need the comment and it's clearer for all of the cases.

**Review by rappoccio — CHANGES_REQUESTED** [review]

**Comment by epalencia** [comment]
Please test

**Comment by epalencia** [comment]
Please test

**Comment by aloeliger** [comment]
please test

The quoted error is with an x509 proxy, which this PR should have no way of affecting. I would imagine/hope the error is temporary

**Comment by epalencia** [comment]
Please test

**Comment by aloeliger** [comment]
+l1

- Good from the L1 side, already reviewed in https://github.com/cms-l1t-offline/cmssw/pull/1170
- Bit pattern changes done.

**Comment by srimanob** [comment]
Hi @SClarkPhysics @aloeliger @rappoccio 
I see a request change by @rappoccio but I can't find what is the quest. When I click on request change button, it just point to the code, but I have no clue what is a request.

**Comment by SClarkPhysics** [comment]
Hi @srimanob , the request was to change the variables from decimal to binary literals. I implemented these changes in this commit: https://github.com/cms-sw/cmssw/pull/43363/commits/a3ea23be982f8875843a5d7ecdc982bef02474e3 

I had to expand a box that said "24 hidden items" in this thread to see the original request

**Comment by srimanob** [comment]
+Upgrade

**Pull request merged**